Adafruit_Arcada.cpp

```diff
@@ -111,6 +111,16 @@ bool Adafruit_Arcada::begin(void) {
   }
 #endif
 
+#ifdef SPIWIFI
+  WiFi.status();
+  delay(100);
+  if (WiFi.status() == WL_NO_MODULE)
+  {
+    _has_wifi = false;
+  } else {
+    _has_wifi = true;
+  }
+#endif
```

Comment:
_has_wifi should be false if theres no SPIWIFI def

Reply:
Latest commit sets `_has_wifi` based on if SPIWIFI has been defined.